lib/fetch/dataURL.js

```diff
@@ -561,6 +564,33 @@ function serializeAMimeType (mimeType) {
   return serialization
 }
 
+/**
+ * @see https://fetch.spec.whatwg.org/#http-whitespace
+ * @param {string} char
+ */
+function isHTTPWhiteSpace (char) {
+  return char === '\r' || char === '\n' || char === '\t' || char === ' '
```

Comment:
Can you compare ascii numbers?

Reply:
I don't think it's needed, or at least I'd prefer to wait for there to be benchmarks before switching over.